src/components/layout/navbar.tsx

```diff
@@ -106,7 +109,7 @@ const NavBar = () => {
             <Box>
               <Tooltip title={strings.header.openUserMenu}>
                 <IconButton onClick={handleOpenUserMenu} sx={{ p: 0 }}>
-                  <Avatar src={loggedInPersonAvatar} />
+                  {/* <Avatar src={loggedInPersonAvatar} /> */}
```

Comment:
Perhaps this could be restored using the UsersAvatar I commented on earlier in a future PR?